dynamodb/src/test/scala/zio/dynamodb/codec/ItemDecoderSpec.scala

```diff
@@ -284,35 +284,20 @@ object ItemDecoderSpec extends ZIOSpecDefault with CodecTestFixtures {
 
       assert(actual)(isRight(equalTo(PreBilled(id = 1, s = "foobar"))))
     },
-    test("decodes case object only enum with @enumOfCaseObjects annotation and without @caseName annotation") {
+    test("decodes case object only enum with @simpleEnum annotation and without @caseName annotation") {
```

Comment:
I'm not pretty sure if we need to keep these test cases as it seems they test what was tested by zio-schema.